proposals/20191030-api.md

```diff
@@ -0,0 +1,118 @@
+# Falco APIs
+
+## Summary
+
+This is a proposal to better structure the Falco API.
+
+It will contain a list of gRPC services that can be added to have a first stable version of the API for v1.0.0.
```

Comment:
So is gRPC the API? or is gRPC the first implementation of a way of exposing the API over a network socket?

I think maybe the API is simply the agreement of the shape of objects, and what they represent and do right?